src/containers/Profile.jsx

```diff
@@ -2,28 +2,20 @@ import { connect } from 'react-redux'
 import compose from 'lodash/flowRight'
 
 import { withClient } from 'cozy-client'
-import { translate } from 'cozy-ui/transpiled/react/I18n'
 import Alerter from 'cozy-ui/transpiled/react/deprecated/Alerter'
 
-import { updateInfo, fetchInfos } from 'actions'
 import { requestExport, fetchExportData } from 'actions/export'
 import { precheckImport, submitImport } from 'actions/import'
 import ProfileView from 'components/Profile/ProfileView'
 
 const mapStateToProps = state => ({
-  fields: state.fields,
   passphrase: state.passphrase,
-  instance: state.instance,
   exportData: state.exportData,
   importData: state.importData,
   isFetching: state.ui.isFetching
 })
 
 const mapDispatchToProps = (dispatch, ownProps) => ({
-  fetchInfos: () => dispatch(fetchInfos(ownProps.client)),
```

Comment:
does the `fetchInfos` is still needed somewhere? Can we delete it?

Reply:
It still used into intent but I will something to tackle in my next PR. I'd like to see if they can be removed as it was used for Claudy button and it was removed from the cozy-bar